contributing.md

```diff
@@ -0,0 +1,45 @@
+# Contributing
+
+Wanna help us on Qwant Maps ? Awesome, thanks for taking the time to contribute ! Here are a few things you can do.
+
+# Improving the map
+Qwant Maps main data source is OpenStreetMap (OSM). It's an international project to create an open geographical database of the entire world: https://www.openstreetmap.org/about
+
+If you know your neighborhood, you are entitled to participate: join the community and improve the map with us.
+You will need a free account here: https://www.openstreetmap.org/user/new
+To get you started, check out:
+* https://www.openstreetmap.org/welcome
+* https://learnosm.org/fr/beginner/
+* https://wiki.openstreetmap.org/wiki/Contribute_map_data
+* [French local chapter website](https://www.openstreetmap.fr/contribuer/) (:fr:)
+
+# Translating
+ :fr:  :it:  :ru:  :de: *Help us bring Qwant Maps in your language*   :jp:  :kr:  :cn:  :es:
+
+You can join the translation team on [transifex](https://www.transifex.com/qwant-1/qwant-maps/translate/), ask for your language and start translating.
+
+# Reporting Bugs or suggesting enhancements
+
+If you find a bug on the map (a missing place or some outdated location), you can fix it by [contributing to OpenStreetMap](https://www.openstreetmap.org/welcome). If you don't want to take the time to sign up and learn how to edit, you can also [add a note on OSM website](https://www.openstreetmap.org/note/new) so other contributors may investigate.
+
+Otherwise, you can [submit an issue](https://github.com/QwantResearch/qwantmaps/issues/new) in this repository. Please provide as much details as you can, with screenshots and steps to reproduce.
+
+# Contributing code and documentation changes
+
+If you want to deep dive and help out with some development on Qwant Maps, then first read along in this repository to understand the architecture and find out some repository to contribute on. [Cosmogony](https://github.com/osm-without-borders/cosmogony) or [Erdapfel](https://github.com/QwantResearch/erdapfel) may be good candidates to get started.
+
+If you have a bugfix or new feature that you would like to contribute to the project, please find or open an issue about it first: maybe we are already working on it, or there are particular issues that you should know about before implementing the change. :pray: Thanks again!
+
+# Expanding
+ :earth_americas: *Help us bring Qwant Maps in your country*  :earth_asia:
+
+Qwant Maps is developed in France and for now mainly tested in France. We need your local knowledge to make it work in the entire world :rocket:
```

Comment:
I'm not sure, but I would nice have said "and for now has been mainly tested in France". (or by french people?)